proposals/devcontainer-distribution.md

```diff
@@ -0,0 +1,71 @@
+# Community creation of templates and features and their distribution.
+
+This proposal is created to arrive at a schema where different communities and organizations can create features and devcontainer templates and make them discoverable to users.
+
+The proposal centers on the following ideas:
+
+- Discoverability for users both on the web (github.com) and on any tools that implement the [Dev container specification](../docs/specs/devcontainer-reference.md).
+- Users can validate the origin of the template or feature that they want to use.
+- Users can pin a particular version of a feature to allow consistent repeatable environments.
+
+# Proposal
+
+## Github Topics
+
+This can be achieved with the use of [GitHub Topics](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics), specifically the [devcontainers](https://github.com/topics/devcontainers) topic. Additionally the repositories can be marked with **devcontainers-templates** and/or **devcontainers-features** to mark specifically the contents of the repository.
+
+Any UI application can leverage this search to provide a list of repositories that contain features or templates and present them for their users. The application would query the latest release and obtain the details to show them to the users.
```

Comment:
The latest release of what - the dev container-related repos? Or something else (i.e. the spec, CLI, etc)? If the former, I'd suggest:

```suggestion
Any UI application can leverage this search to provide a list of repositories that contain features or templates and present them to their users. The application would query the latest release of repos within the list and obtain their details to show them to the users.
```